opensrp-path/src/main/java/org/smartregister/path/watchers/HIA2ReportFormTextWatcher.java

```diff
@@ -0,0 +1,89 @@
+package org.smartregister.path.watchers;
+
+import android.text.Editable;
+import android.text.TextWatcher;
+
+import com.rengwuxian.materialedittext.MaterialEditText;
+import com.vijay.jsonwizard.fragments.JsonFormFragment;
+
+import org.smartregister.path.service.HIA2Service;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class HIA2ReportFormTextWatcher implements TextWatcher {
+
+    private static final Map<String, String[]> aggregateFieldsMap;
+
+    static {
+        aggregateFieldsMap = new HashMap<>();
+        aggregateFieldsMap.put(HIA2Service.CHN1_011, new String[]{HIA2Service.CHN1_005, HIA2Service.CHN1_010});
```

Comment:
There a lot of duplication here, but how do we feel about something like:

```java
String[][] keyValues = { { HIA2Service.CHN1_011, HIA2Service.CHN1_005, HIA2Service.CHN1_010 },
                         { HIA2Service.CHN1_021, HIA2Service.CHN1_015, HIA2Service.CHN1_020 },
                         ...                                                                  };

fore (entry : keyValues) {
    aggregateFieldsMap.put(entry[0], new String[]{entry[1], entry[2]});
}
```

The magic numbers aren't nice, maybe there's a better way to do it without those